Agreements/Internal/7_IoM.txt

```diff
@@ -0,0 +1 @@
+COPX:*:*:PENIL:EIDW:*:London S7:Isle of Man:*:29000:PENIL
```

Comment:
Think this agreement should be |PENIL (descending), though this might need updating on the agreed levels diagrams too